modules/reference/pages/console/record-deserialization.adoc

```diff
@@ -1,27 +1,35 @@
-= Record Deserialization
-:description: Learn how record deserialization in Redpanda Console works and what encodings it supports.
+= Deserialization in Redpanda Console
+:description: Learn how Redpanda Console deserializes messages.
 :page-aliases: console:features/record-deserialization.adoc
 
-When consuming messages, Redpanda Console's message viewer is capable of automatically deserializing messages.
-It identifies the correct deserialization type by trying to decode the record's key, value, or header with all
-available deserialization methods. The supported deserialization methods are:
+In Redpanda, the messages exchanged between producers and consumers contain raw bytes. Schemas work as an agreed-upon format, like a contract, for producers and consumers to serialize and deserialize those messages. If a producer breaks this contract, consumers can fail. 
 
-* Kafka's internal binary formats (for example the `__consumer_offsets` topic )
+Redpanda Console automatically tries to deserialize incoming messages and show them in human-readable format. It tries different deserializers and lets you choose the deserializer before producing the message in the desired format. If the deserialization fails, it provides troubleshooting information. For example, deserialization requires fetching schemas from a schema registry, and that fetch could fail. If downstream applications crash, you can find which message caused the crash, check the message content, and see the schema that was used in deserializing it. You can also download the raw bytes of the message to feed it directly to your client deserializer or share it with support.
+
+== Automatically deserialize messages
+
+Redpanda Console tries to automatically identify the correct deserialization type by decoding the message's key, value, or header with all available deserialization methods. The following deserialization methods are supported:
+
+* Kafka's internal binary formats; for example, the `__consumer_offsets` topic
 * JSON
-* JSON Schema with schema registry encoding
+* JSON with schema registry encoding
 * XML
 * Avro with schema registry encoding
 * Protobuf
 * Protobuf with schema registry encoding
-* Messagepack (only for topics that are explicitly enabled to test MessagePack)
-* UTF-8 / Strings
+* Messagepack (for topics explicitly enabled to test MessagePack)
+* UTF-8 / strings
```

Comment:
Smile and uint (8, 16, 32 and 64bit) are missing on that list

Reply:
Thanks @weeco! Please confirm if this is correct: https://deploy-preview-175--redpanda-docs-preview.netlify.app/23.3/reference/console/record-deserialization/#automatic-deserialization

Reply:
Yes looks good to me 👍